diff --git a/base/BUILD.gn b/base/BUILD.gn
index d4eac5820262eea7854feb6d4d850481cef05510..5c84007ec402fe7b2470bf94b35e14325cd3c443 100644
--- a/base/BUILD.gn
+++ b/base/BUILD.gn
@@ -4349,6 +4349,7 @@ if (is_android) {
       "android/java/src/org/chromium/base/task/ThreadPoolTaskExecutor.java",
       "android/java/src/org/chromium/base/task/UiThreadTaskExecutor.java",
     ]
+    sources += brave_java_base_sources
 
     if (!is_cronet_build) {
       sources += [ "android/java/src/org/chromium/base/IntentUtils.java" ]
