diff --git a/build/android/gyp/dex.py b/build/android/gyp/dex.py
index ce91b38b1ae3bf0be4046b33a3efd5f0ed0f72d1..91f4d597a28852fd67145a7bf699820585950005 100755
--- a/build/android/gyp/dex.py
+++ b/build/android/gyp/dex.py
@@ -36,7 +36,7 @@ INTERFACE_DESUGARING_WARNINGS = (r'default or static interface methods', )
 _SKIPPED_CLASS_FILE_NAMES = (
     'module-info.class',  # Explicitly skipped by r8/utils/FileUtils#isClassFile
 )
-
+_IGNORE_WARNINGS += ( r'.Brave', )
 
 def _ParseArgs(args):
   args = build_utils.ExpandFileArgs(args)
