diff --git a/chrome/browser/BUILD.gn b/chrome/browser/BUILD.gn
index ae812091f7ffffd5b3ac176f0539f7f4830cfb6e..255b5a17dd6efe160cc1ea918a5e698068ff3a00 100644
--- a/chrome/browser/BUILD.gn
+++ b/chrome/browser/BUILD.gn
@@ -2520,6 +2520,7 @@ static_library("browser") {
       "metrics/family_link_user_metrics_provider.h",
     ]
   }
+  import("//brave/browser/sources.gni") sources += brave_chrome_browser_sources deps += brave_chrome_browser_deps public_deps += brave_chrome_browser_public_deps allow_circular_includes_from += brave_chrome_browser_allow_circular_includes_from
 
   if (is_android) {
     sources += [
