diff --git a/chrome/browser/ui/views/tabs/tab_strip.cc b/chrome/browser/ui/views/tabs/tab_strip.cc
index eb6452cb92c932140a43c5d265bd577d1cfa8026..2ab3a6f1a1ba0f0f36c3bd52bbab7263499ed7d7 100644
--- a/chrome/browser/ui/views/tabs/tab_strip.cc
+++ b/chrome/browser/ui/views/tabs/tab_strip.cc
@@ -134,11 +134,6 @@ std::unique_ptr<TabContainer> MakeTabContainer(
     TabStrip* tab_strip,
     TabHoverCardController* hover_card_controller,
     TabDragContext* drag_context) {
-  if (base::FeatureList::IsEnabled(features::kSplitTabStrip)) {
-    return std::make_unique<CompoundTabContainer>(
-        raw_ref<TabContainerController>(*tab_strip), hover_card_controller,
-        drag_context, *tab_strip, tab_strip);
-  }
   return std::make_unique<TabContainerImpl>(
       *tab_strip, hover_card_controller, drag_context, *tab_strip, tab_strip);
 }
@@ -489,6 +484,7 @@ class TabStrip::TabDragContextImpl : public TabDragContext,
 
   std::vector<gfx::Rect> CalculateBoundsForDraggedViews(
       const std::vector<TabSlotView*>& views) override {
+    BRAVE_CALCULATE_BOUNDS_FOR_DRAGGED_VIEWS
     DCHECK(!views.empty());
 
     std::vector<gfx::Rect> bounds;
@@ -732,6 +728,7 @@ class TabStrip::TabDragContextImpl : public TabDragContext,
         continue;
       }
 
+      BRAVE_CALCULATE_INSERTION_INDEX
       // If there's a group header here, and we're dragging a group, we might
       // end up on either side of that header. Check both cases to find the
       // best option.
@@ -885,7 +882,7 @@ END_METADATA
 
 TabStrip::TabStrip(std::unique_ptr<TabStripController> controller)
     : controller_(std::move(controller)),
-      hover_card_controller_(std::make_unique<TabHoverCardController>(this)),
+      hover_card_controller_(std::make_unique<BraveTabHoverCardController>(this)),
       drag_context_(*AddChildView(std::make_unique<TabDragContextImpl>(this))),
       tab_container_(
           *AddChildViewAt(MakeTabContainer(this,
