diff --git a/chrome/browser/ui/webui/BUILD.gn b/chrome/browser/ui/webui/BUILD.gn
index 05b29776675d0a033e526d055503fdd9185d6cb8..06b2cb26df9e8df57669f1e62859169f135da424 100644
--- a/chrome/browser/ui/webui/BUILD.gn
+++ b/chrome/browser/ui/webui/BUILD.gn
@@ -76,4 +76,5 @@ source_set("configs") {
       deps += [ "//ash/webui/sample_system_web_app_ui" ]
     }
   }
+  import("//brave/browser/ui/webui/sources.gni") deps += brave_browser_ui_webui_configs_deps
 }
