diff --git a/chrome/browser/ui/webui/extensions/extensions_ui.cc b/chrome/browser/ui/webui/extensions/extensions_ui.cc
index 211a3f750209ce320f1526ddce1077ebce51caf9..7c59a32b06ae86f50ac3a6906066917a9be446ea 100644
--- a/chrome/browser/ui/webui/extensions/extensions_ui.cc
+++ b/chrome/browser/ui/webui/extensions/extensions_ui.cc
@@ -411,6 +411,7 @@ content::WebUIDataSource* CreateAndAddExtensionsSource(Profile* profile,
       base::FeatureList::IsEnabled(
           extensions_features::kExtensionsMenuAccessControlWithPermittedSites));
 
+  BRAVE_CREATE_EXTENSIONS_SOURCE
   return source;
 }
 
