diff --git a/chrome/browser/ui/views/permissions/chip_controller.cc b/chrome/browser/ui/views/permissions/chip_controller.cc
index eeb81ac26534db1487e7e8a3f40fc13884002670..8b23b62229a9b2a38a0c9085e7d1c6982d392d94 100644
--- a/chrome/browser/ui/views/permissions/chip_controller.cc
+++ b/chrome/browser/ui/views/permissions/chip_controller.cc
@@ -435,6 +435,7 @@ void ChipController::HideChip() {
   chip_->SetVisible(false);
   // When the chip visibility changed from visible -> hidden, the locationbar
   // layout should be updated.
+  BRAVE_CHIP_CONTROLLER_HIDE_CHIP
   GetLocationBarView()->InvalidateLayout();
 }
 
