diff --git a/chrome/browser/ui/views/permissions/permission_prompt_bubble_view.cc b/chrome/browser/ui/views/permissions/permission_prompt_bubble_view.cc
index c32ff04da4296cc4143b8e663c1375040952b90a..4ca639c46675c5e7332ea82aaba6b77964785f67 100644
--- a/chrome/browser/ui/views/permissions/permission_prompt_bubble_view.cc
+++ b/chrome/browser/ui/views/permissions/permission_prompt_bubble_view.cc
@@ -218,6 +218,7 @@ PermissionPromptBubbleView::PermissionPromptBubbleView(
        GetVisibleRequests(*delegate.get())) {
     AddRequestLine(request);
   }
+  BRAVE_PERMISSION_PROMPT_BUBBLE_VIEW
 
   absl::optional<std::u16string> extra_text = GetExtraText(*delegate.get());
   if (extra_text.has_value()) {
