diff --git a/components/permissions/permission_uma_util.cc b/components/permissions/permission_uma_util.cc
index 267eb570e5dc72d70818e02755363d9c2690f854..2e67819337da8fb8579710cfa63d4aa7e0e69f1a 100644
--- a/components/permissions/permission_uma_util.cc
+++ b/components/permissions/permission_uma_util.cc
@@ -117,6 +117,7 @@ RequestTypeForUma GetUmaValueForRequestType(RequestType request_type) {
 #endif
     case RequestType::kTopLevelStorageAccess:
       return RequestTypeForUma::PERMISSION_TOP_LEVEL_STORAGE_ACCESS;
+    BRAVE_GET_UMA_VALUE_FOR_REQUEST_TYPE
   }
 }
 
