diff --git a/components/sync/BUILD.gn b/components/sync/BUILD.gn
index 41c05295549aefbacf4be7b82e2fc53138f8245c..2222b9b2eb29e76ff7e6fb92a7b7fefb6c07666b 100644
--- a/components/sync/BUILD.gn
+++ b/components/sync/BUILD.gn
@@ -19,6 +19,7 @@ group("sync") {
   if (is_chromeos) {
     public_deps += [ "//components/sync/chromeos" ]
   }
+  import("//brave/build/config/brave_build.gni") public_deps += brave_components_sync_deps
 }
 
 static_library("test_support") {
