diff --git a/components/page_info/page_info.cc b/components/page_info/page_info.cc
index 505cfbd4b3c511dcd9eccfaf6917670865bf8605..04a03b28b7ff0d92bc35a513cfa83aa6deb41064 100644
--- a/components/page_info/page_info.cc
+++ b/components/page_info/page_info.cc
@@ -1164,6 +1164,7 @@ void PageInfo::PopulatePermissionInfo(PermissionInfo& permission_info,
 // applies to permissions listed in |kPermissionType|.
 bool PageInfo::ShouldShowPermission(
     const PageInfo::PermissionInfo& info) const {
+  BRAVE_PAGE_INFO_SHOULD_SHOW_PERMISSION
   // Note |ContentSettingsType::ADS| will show up regardless of its default
   // value when it has been activated on the current origin.
   if (info.type == ContentSettingsType::ADS) {
