diff --git a/ios/chrome/app/BUILD.gn b/ios/chrome/app/BUILD.gn
index 62ed2c755550f8d1fdd558eb0f4554e1cf410fe2..c8aa98c8148a468589871783f1658c8e2c7295be 100644
--- a/ios/chrome/app/BUILD.gn
+++ b/ios/chrome/app/BUILD.gn
@@ -728,8 +728,7 @@ ios_app_bundle("chrome") {
   # (see https://crbug.com/1394553 for details of the build failure). This
   # is a stop-gap solution.
   if (target_environment == "catalyst" ||
-      ios_provider_target !=
-      "//ios/chrome/browser/providers:chromium_providers") {
+      !ios_chrome_links_with_lottie_framework) {
     assert_no_deps += [
       "//ios/third_party/lottie:lottie+bundle",
       "//ios/third_party/lottie:lottie+link",
