diff --git a/third_party/blink/renderer/modules/mediastream/media_devices.cc b/third_party/blink/renderer/modules/mediastream/media_devices.cc
index 67fb73f369e3613c2fd177794d90a020587f230e..0a0d3761268a1ccfd13180278ab3b0c26af98b5d 100644
--- a/third_party/blink/renderer/modules/mediastream/media_devices.cc
+++ b/third_party/blink/renderer/modules/mediastream/media_devices.cc
@@ -1000,6 +1000,7 @@ void MediaDevices::DevicesEnumerated(
     }
   }
 
+  BRAVE_MEDIA_DEVICES_DEVICES_ENUMERATED
   RecordEnumeratedDevices(result_tracker->GetScriptState(), media_devices);
   // TODO(crbug.com/1395324): Remove this custom EnumerateDevices latency
   // tracking by reverting crrev.com/c/3944912/ once the
