diff --git a/third_party/blink/renderer/core/script/script_loader.cc b/third_party/blink/renderer/core/script/script_loader.cc
index 64d1b67b37007f00fc8914f81526cb2979d794d2..1bf311a571b3c8fd7f3e330793c1d0b6237f8fcd 100644
--- a/third_party/blink/renderer/core/script/script_loader.cc
+++ b/third_party/blink/renderer/core/script/script_loader.cc
@@ -564,6 +564,7 @@ PendingScript* ScriptLoader::PrepareScript(
       break;
 
     case ScriptTypeAtPrepare::kWebBundle:
+      BRAVE_SCRIPT_LOADER_PREPARE_SCRIPT_WEB_BUNDLE
     case ScriptTypeAtPrepare::kClassic:
     case ScriptTypeAtPrepare::kModule:
     case ScriptTypeAtPrepare::kImportMap:
@@ -713,6 +714,7 @@ PendingScript* ScriptLoader::PrepareScript(
                              parser_state, credentials_mode, referrer_policy,
                              fetch_priority_hint, render_blocking_behavior,
                              RejectCoepUnsafeNone(false));
+  BRAVE_SCRIPT_LOADER_PREPARE_SCRIPT
 
   // <spec step="28">Let settings object be el's node document's relevant
   // settings object.</spec>
