diff --git a/src/codegen/compiler.cc b/src/codegen/compiler.cc
index 66cff887179ad9e37749481f9fb9e368961b16f0..8d32e26befbe407391d9022c8f690525c83bca5c 100644
--- a/src/codegen/compiler.cc
+++ b/src/codegen/compiler.cc
@@ -2901,6 +2901,7 @@ MaybeHandle<JSFunction> Compiler::GetFunctionFromEval(
     }
   }
   DCHECK(is_compiled_scope.is_compiled());
+  BRAVE_COMPILER_GET_FUNCTION_FROM_EVAL
 
   return result;
 }
